### DISCUSSION TITLE
Как в быстрой форме регистрации проверить логин?

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#427] [Как это сделать?] [answered]
В быстрой форме регистрации нужно проверить, существует ли такой логин.  Если да, то не дать форме сохраниться.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. добавьте regexp аналогичный полю логин в табл users `^[a-z0-9_\-.]{3,30}$`

2. в кнопке сохранения первым действием возьмите select по этому логину из таблицы users и если он не пустой (логин есть) то выдайте errorException — это остановит все последующие действия
